packages/runtime-common/query.ts

```diff
@@ -419,3 +421,16 @@ function assertRangeFilter(
     });
   });
 }
+
+export function assertKey(key: string, pointer: string[]) {
+  if (key.startsWith('[') && key.endsWith(']')) {
+    throw new Error(
+      `${pointer.join('/')}: field names cannot be wrapped in brackets: ${key}`,
+    );
+  }
+}
+
+export const parseQuery = (queryString: string) => {
+  // @ts-ignore
```

Comment:
let's be a little more precise why we are ignoring. I usually do:
```
// @ts-expect-error no types available for XXX
```
this way when types do become available our lint will let us know that the following line doesn't need to be ignored anymore

Reply:
Thanks. Just fixed

Reply:
This has been merged https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71087 and deployed on @6.9.17 of qs. No need for `ts-expect-error` anymore